applications/GeoMechanicsApplication/tests/cpp_tests/test_fluid_pressure.cpp

```diff
@@ -0,0 +1,36 @@
+// KRATOS___
+//     //   ) )
+//    //         ___      ___
+//   //  ____  //___) ) //   ) )
+//  //    / / //       //   / /
+// ((____/ / ((____   ((___/ /  MECHANICS
+//
+//  License:         geo_mechanics_application/license.txt
+//
+//  Main authors:    Gennady Markelov
+//
+
+#include "boost/numeric/ublas/assignment.hpp"
+#include "custom_retention/retention_law.h"
+#include "includes/checks.h"
+#include "testing/testing.h"
+
+using namespace Kratos;
+
+namespace Kratos::Testing
+{
+
+KRATOS_TEST_CASE_IN_SUITE(CalculateFluidPressureGivesCorrectResults, KratosGeoMechanicsFastSuite)
+{
+    Vector N(5);
+    N <<= 1.0, 2.0, 3.0, 4.0, 5.0;
```

Comment:
For an real interpolation function based on 5 nodes, these values are strange. However, for testing they are fine.

Reply:
It is just for testing. Put something into and to get this later back.

Reply:
No need to change.